packages/hadron-build/lib/target.js

```diff
@@ -624,7 +633,7 @@ class Target {
       name: this.productName
     });
 
-    const debianVersion = this.version.replace(/\-/g, '~');
```

Comment:
This doesn't seem necessary based on [deb version policies](https://www.debian.org/doc/debian-policy/ch-controlfields.html#version) and was removed when uploading release assets to github anyway, so seemed like a good idea to remove this. Please tell me if you think otherwise

Reply:
yeah, looks like we don't do that for mongosh